Assets/MixedRealityToolkit/_Core/Devices/MixedReality/WindowsMixedRealityController.cs

```diff
@@ -138,7 +138,7 @@ private void UpdatePointerData(InteractionSourceState interactionSourceState, Mi
             interactionSourceState.sourcePose.TryGetPosition(out currentPointerPosition, InteractionSourceNode.Pointer);
             interactionSourceState.sourcePose.TryGetRotation(out currentPointerRotation, InteractionSourceNode.Pointer);
 
-            if (CameraCache.Main.transform.parent != null)
+            if (CameraCache.Main?.transform.parent != null)
```

Comment:
This should never be null, unless you forgotten to tag your camera as `MainCamera`.
![image](https://user-images.githubusercontent.com/13334553/42177092-44054cea-7de0-11e8-921a-22031c25f1b1.png)

Reply:
Which is easy for developers to do. I like having this check.

Reply:
Let's put an assert in CameraCache.Main telling people to tag their camera properly.

Reply:
That's the real fix.  This null check should never have to happen.